### PR TITLE
Upgrade to .NET Core 3.1.2 and update packages to their latest version

### PR DESCRIPTION
**Pull request by pmotameni**
Updated to .NET Core 3.1.2
Updated Swashbuckle from the preview version to the latest
Some cleanups

### CODE REVIEW

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -103,5 +84,34 @@ private static IMapper GenerateMapperConfiguration(IEnumerable<string> assemblyN
...
+
+        private static void SetDocumentPath(SwaggerConfig swaggerConfig, SwaggerGenOptions options)
+        {
+            // Set the comments path for the Swagger JSON and UI.
+            var xmlFile = swaggerConfig.XmlFile;
+            var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+            options.IncludeXmlComments(xmlPath);
+        }
+
+        private static void AddSecurityDefinition(SwaggerGenOptions oprions) =>
```
Comment (andykongchi) [inline src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: spelling: `options`

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -103,5 +84,34 @@ private static IMapper GenerateMapperConfiguration(IEnumerable<string> assemblyN
...
+            oprions.AddSecurityDefinition("Bearer", new OpenApiSecurityScheme()
+            {
+                Name = "Authorization",
+                In = ParameterLocation.Header,
+                Type = SecuritySchemeType.ApiKey,
+                Description = "Please insert JWT with Bearer into field",
+            });
+
+        private static void AddSecurityRequirement(SwaggerGenOptions oprions) =>
```
Comment (andykongchi) [inline src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: spelling: `options`

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs (hunk 1 above)
Comment (pmotameni) [reply src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: Fixed!

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs (hunk 2 above)
Comment (pmotameni) [reply src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: Fixed!

**Pull request merged**